plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -20,9 +20,14 @@ public static void main(String[] args) throws IOException {
             throw new IOException("Failed to read command-line arguments", e);
         }
 
-        Config cfg = options.setup();
-        PluginManager pm = new PluginManager(cfg);
-        pm.start();
+        if (options.isShowVersion()) {
+            options.showVersion();
```

Comment:
I would get rid of the else statement and exit here, also means if anyone adds any code after the else later it won't accidentally get executed
```suggestion
            options.showVersion();
            System.exit(0);
```